samigo/samigo-hibernate/src/java/org/sakaiproject/tool/assessment/data/dao/questionpool/QuestionPoolData.hbm.xml

```diff
@@ -11,7 +11,7 @@
       </generator>
     </id>
     <property name="title" type="string" length="255" column="TITLE" not-null="false" />
-    <property name="description" type="string" length="255" column="DESCRIPTION" not-null="false" />
+    <property name="description" type="text" length="255" column="DESCRIPTION" not-null="false" />
```

Comment:
Probably should remove the length field here as text is 65K size and this is misleading.